restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AccountsEndpoint.java

```diff
@@ -235,6 +249,23 @@ protected Response putAccount(final MultivaluedMap<String, String> data, final M
                     account = account.setRole(parent.getRole());
                 }
                 dao.addAccount(account);
+
+                // Create default SIP client data
+                MultivaluedMap<String, String> clientData = new MultivaluedMapImpl();
+                clientData.add("Login", data.getFirst("EmailAddress"));
```

Comment:
The work on Restcomm organizations will complement that here and will make sure we have unique sip clients